### PR TITLE
Revise notification messages

### PR DESCRIPTION
**Pull request by bp323**
I grepped the codebase for all notifications we have, these are in no particular order.
The first bit before the ',' is the notification title, the second the notification body.

- [x] 'Login failed', 'Logging in to the application failed'
- [x] 'Configuration not retrieved.', 'The configuration could not be successfully retrieved.'
- [x] 'Fetching tenants failed.', 'An error occurred while fetching the tenants.'
- [x] 'Fetching apps failed.', 'An error occurred while fetching the apps.'
- [x] 'Fetching admins failed.', 'An error occurred while fetching the admins.'
- [x] 'App not created.', 'The app could not be successfully created.'
- [x] 'App created.', 'The app was successfully created.'
- [x] 'App not updated.', 'The app could not be successfully updated.'
- [x] 'App updated.', 'The app was successfully updated.'
- [x] 'Tenant not created.', 'The tenant could not be successfully created.'
- [x] 'Tenant updated.', 'The tenant was successfully updated.'
- [x] 'Administrator not created.', 'The administrator could not be successfully created.'
- [x] 'Administrator created.', 'The administrator was successfully created.'
- [x] 'Administrator not updated.', 'The administrator could not be successfully updated.'
- [x] 'Administrator updated.', 'The administrator was successfully updated.'
- [x] 'Configuration not updated.', 'The configuration could not be successfully updated.'
- [x] 'Configuration updated.', 'The configuration was successfully updated.'
- [x] 'Fetching triposes failed.', 'An error occurred while fetching the triposes.'
- [x] 'Login failed', 'Logging in to the application failed'
- [x] 'Logout failed', 'Logging out of the application failed'
- [x] 'Login failed', 'Logging in to the application failed'
- [x] 'Fetching triposes failed.', 'An error occurred while fetching the triposes.'
- [x] 'App not configured', 'The application has not yet been properly configured for an academic year.', 'error', null, true);
- [x] 'Series title not updated.', 'The series title could not be successfully updated.'
- [x] 'Series title updated.', 'The series title was successfully updated.');
- [x] 'Events not updated.', 'Not all events could be successfully updated.'
- [x] 'Events updated.', 'The events where successfully updated.');
- [x] 'Series not retrieved.', 'The event series could not be successfully retrieved.'
- [x] 'Events not retrieved.', 'The events could not be successfully retrieved.'
- [x] 'Series not borrowed.', 'The series could not be successfully borrowed.'
- [x] 'Series borrowed.', 'The series were successfully borrowed.'
- [x] 'Fetching user calendar failed.', 'An error occurred while fetching the user calendar.'
- [x] 'Removing series failed.', 'An error occurred while removing the series.'
- [x] 'Series removed.', 'The series was successfully removed.');
- [x] 'Deleting series failed.', 'An error occurred while deleting the series.'
- [x] 'Series deleted.', 'The series was successfully deleted.');
- [x] 'Retrieving module failed.', 'An error occurred while retrieving the module information.'
- [x] 'Retrieving module failed.', 'An error occurred while retrieving the module information.'
- [x] 'Fetching series failed.', 'An error occurred while fetching the series information.'
- [x] 'Fetching module failed.', 'An error occurred while fetching the module information.'
- [x] 'Fetching modules failed.', 'An error occurred while fetching the modules.'
- [x] 'Module not created.', 'The module could not be successfully created.'
- [x] 'Module created.', 'The module was successfully created.'
- [x] 'Series not created.', 'The series could not be successfully created.'
- [x] 'Series not created.', 'The series could not be successfully created.'
- [x] 'Series created.', 'The series was successfully created.'
- [x] 'Events not added.', 'The events could not be successfully added to your calendar.'
- [x] 'Events added.', 'All events where successfully added to your calendar.', 'success', 'notification-events-added');
- [x] 'Events not removed.', 'The events could not be successfully removed from your calendar.'
- [x] 'Events removed.', 'The events were successfully removed from your calendar.', 'success', 'notification-events-removed');
- [x] 'Events not added.', 'The events could not be successfully added to your calendar.'
- [x] 'Events added.', 'All events where successfully added to your calendar.', 'success', 'notification-events-added');
- [x] 'Event not removed.', 'The event could not be successfully removed from your calendar.'
- [x] 'Event removed.', 'The event was successfully removed from your calendar.', 'success', 'notification-events-removed');
- [x] 'Request error.', 'An error occurred while requesting the unit information.'
- [x] 'Publishing failed.', 'The part could not be successfully published.'
- [x] '"' + orgUnit.displayName + '" has been published.', 'Students and Lecturers can now access the information in the timetable.');
- [x] 'Request error.', 'An error occurred while requesting the unit information.'

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: shared/gh/js/views/gh.admin-batch-edit.js [hunk 1]
```diff
@@ -414,7 +414,7 @@ define(['gh.core', 'gh.constants', 'gh.utils', 'moment', 'gh.calendar', 'gh.admi
         if (!triposData) {
             utils.getTriposStructure(function(err, data) {
                 if (err) {
-                    return utils.notification('Fetching triposes failed.', 'An error occurred while fetching the triposes.', 'error');
+                    return gh.utils.notification('Could not fetch triposes', constants.messaging.default.error, 'error');
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit.js]: We might want to use `gh.utils` throughout the entire code to be consistent (and remove `utils` form the requires above).

**Comment by Coenego** [comment]
Left a suggestion for @bp323

**Comment by bp323** [comment]
Followed up on review, back to @Coenego

**Pull request merged**